impl/protobuf-es/runner.ts

```diff
@@ -15,38 +15,50 @@
 // limitations under the License.
 
 import {
-  ConformanceRequest,
-  ConformanceResponse,
-  FailureSet,
+  type ConformanceRequest,
+  ConformanceRequestSchema,
+  type ConformanceResponse,
+  ConformanceResponseSchema,
+  FailureSetSchema,
   TestCategory,
   WireFormat,
 } from "./gen/conformance/conformance_pb.js";
-import { TestAllTypesProto3 } from "./gen/google/protobuf/test_messages_proto3_pb.js";
-import { extension_int32, TestAllTypesProto2 } from "./gen/google/protobuf/test_messages_proto2_pb.js";
+import { TestAllTypesProto3Schema } from "./gen/google/protobuf/test_messages_proto3_pb.js";
+import {
+  extension_int32,
+  TestAllTypesProto2Schema,
+} from "./gen/google/protobuf/test_messages_proto2_pb.js";
 import { readSync, writeSync } from "fs";
 import {
-  Any,
+  AnySchema,
+  DurationSchema,
+  FieldMaskSchema,
+  Int32ValueSchema,
+  StructSchema,
+  TimestampSchema,
+  ValueSchema,
+} from "@bufbuild/protobuf/wkt";
+import {
   createRegistry,
-  Duration,
-  FieldMask,
-  Int32Value,
-  Message,
-  Struct,
-  Timestamp,
-  Value,
+  create,
+  toBinary,
+  fromBinary,
+  toJsonString,
+  fromJsonString,
+  type Message,
 } from "@bufbuild/protobuf";
 
 const registry = createRegistry(
-  Value,
-  Struct,
-  FieldMask,
-  Timestamp,
-  Duration,
-  Int32Value,
-  TestAllTypesProto3,
-  TestAllTypesProto2,
-  Any,
-  extension_int32
+  ValueSchema,
+  StructSchema,
+  FieldMaskSchema,
+  TimestampSchema,
+  DurationSchema,
+  Int32ValueSchema,
+  TestAllTypesProto3Schema,
+  TestAllTypesProto2Schema,
+  AnySchema,
+  extension_int32,
```

Comment:
Did you take a look at the implementation in protobuf-es? 

https://github.com/bufbuild/protobuf-es/blob/main/packages/protobuf-conformance/src/conformance.ts

We don't need to cover editions yet, but it would be helpful for debugging if the implementations are similar, and it's a bit more clean to [use the file descriptors](https://github.com/bufbuild/protobuf-es/blob/9d2c0c4cd617e59f422b5b23e2cb29044244b12e/packages/protobuf-conformance/src/conformance.ts#L50) for the registry (no worrying whether you forgot a type, fewer imports).

Reply:
Got it. No, I didn't even look at that implementation. Didn't know about importing file descriptors like that. Updated the runner to use that example (minus editions).